portal/src/components/roles-and-groups/list/RolesAndGroupsBaseList.module.css

```diff
@@ -0,0 +1,10 @@
+.listWrapper {
+  flex: 1 0 0%;
```

Comment:
Use `@apply flex-1` which is `flex: 1 1 0`. The `%` unit doesn't work well in some cases, as mentioned in https://github.com/authgear/authgear-server/blob/roles-and-groups-portal/portal/tailwind.config.js#L14

Reply:
Saw this file was removed, replaced with comment https://github.com/authgear/authgear-server/pull/3994#discussion_r1525862368